zerver/forms.py

```diff
@@ -331,7 +331,7 @@ def clean_email(self) -> str:
 
 
 class ImportRealmOwnerSelectionForm(forms.Form):
-    email = forms.EmailField(validators=[email_not_system_bot])
+    email = forms.EmailField()
```

Comment:
This change is wrong, but I was thinking we want this to just be a user ID anyway and just didn't find time to try changing that, so it's a TODO comment.

Reply:
Changed to use `ID`.